scripts/utilities.js

```diff
@@ -0,0 +1,10 @@
+const replace = require("lodash/replace");
+const lowerCase = require("lodash/lowerCase");
```

Comment:
I think we should stick to native methods where possible. Also let's remove `lodash` from dependencies and just add [lodash.get](https://www.npmjs.com/package/lodash.get) since I think we're only using that utility.